event-handler/state.go

```diff
@@ -17,117 +17,113 @@ limitations under the License.
 package main
 
 import (
+	"encoding/binary"
 	"net"
+	"os"
+	"path"
 	"strings"
 	"time"
 
+	"github.com/gravitational/teleport-plugins/event-handler/lib"
+	"github.com/gravitational/teleport-plugins/lib/logger"
 	"github.com/gravitational/trace"
 	"github.com/peterbourgon/diskv"
-	log "github.com/sirupsen/logrus"
 )
 
 const (
 	// cacheSizeMaxBytes max memory cache
 	cacheSizeMaxBytes = 1024
-)
-
-// State is the state repository
-type State struct {
-	// dv is a diskv instance
-	dv *diskv.Diskv
-
-	// prefix is the state variable prefix
-	prefix string
 
 	// startTimeName is the start time variable name
-	startTimeName string
+	startTimeName = "start_time"
 
 	// cursorName is the cursor variable name
-	cursorName string
+	cursorName = "cursor"
 
 	// idName is the id variable name
-	idName string
+	idName = "id"
+
+	// sessionPrefix is the session key prefix
+	sessionPrefix = "session"
+
+	// storageDirPerms is storage directory permissions when created
+	storageDirPerms = 0755
+)
+
+// State is the state repository
+type State struct {
+	// dv is a diskv instance
```

Comment:
What does this actually imply?

Reply:
It is the struct for managing plugin persistent state as a kvs database. Changed the comment.